utils/llm.py

```diff
@@ -112,18 +105,14 @@ def ask_llm(
     context_length: int = None,
     cost_callback: Callable[[float], None] = None,
     timeout: float = 300,
-    max_tokens: Optional[int] = None,
+    max_response_tokens: int = 1000,
```

Comment:
Is there any reasoning behind this default value? `None` means that it will be allowed to generate until it fills up the context or whatever the API says.

Reply:
Oh I didn't know that one. I thought I needed some ints there.